docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: '3.9'
+
+services:
+  ffc-pay-event-azurite:
+    image: mcr.microsoft.com/azure-storage/azurite
+    command: "azurite --blobHost 0.0.0.0 --queueHost 0.0.0.0 --tableHost 0.0.0.0 --loose"
+    volumes:
+      - azurite_data:/data
+    ports:
+      - "10010:10000"
```

Comment:
Could we map these ports to `100013-10015` just because the file consumer repo uses this range so they can all be run together